README.md

```diff
@@ -1 +1,173 @@
-# Budget-app-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Budget app project <a name="about-project"></a>
+<br>
+This Ruby on Rails project is about a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.<br>
+<br>
+
+## 🛠 Built With <a name="built-with"></a><br>
+
+### Tech Stack
+
+This project is built using the following technologies:
+
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+<br>
+
+#### The app allows the user to:
+
+- **Register and log in, so that the data is private to them.**
+- **Introduce new transactions associated with a category.**
+- **See the money spent on each category**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo & Video presentation<a name="live-demo"></a> <a name='video'></a>
+<br>
+
+- [Check the live demo, mobile version](https://dugetapp.onrender.com)
```

Comment:
Nice job so far😉. Your project looks great👍🏼. But as a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md) for this milestone, your video presentation should contain a piece of code or a feature in the application that you are proud of😉. Your video presentation looks great, but it seems you have forgotten to talk about this. kindly do well to add this👍🏼😉.

<img width="1076" alt="image" src="https://github.com/agneta1991/Budget-app-capstone/assets/64643995/e0b62086-64fd-4a66-80a8-271093820ca8">